configs/s2ef/all/base.yml

```diff
@@ -21,3 +21,8 @@ task:
   grad_input: atomic forces
   train_on_free_atoms: True
   eval_on_free_atoms: True
+  atomwise_metric_atoms:
```

Comment:
It's fine to push merge this, but a slightly better design would be to not require users to remember the atomic numbers. Instead, have a static mapping somewhere and users only have to provide the atomic numbers.

Reply:
Ok, I've updated it to take a list of atomic numbers instead.